docs/proposals/network-policy.md

```diff
@@ -58,12 +56,18 @@ The following objects will be defined on a Namespace Spec.
 
 ```go
 type IngressIsolationPolicy string
+type EgressIsolationPolicy string
 
 const (
 	// Deny all ingress traffic to pods in this namespace. Ingress means 
 	// any incoming traffic to pods, whether that be from other pods within this namespace
 	// or any source outside of this namespace.
 	DefaultDeny IngressIsolationPolicy = "DefaultDeny"
+
+	// Deny all egress traffic from pods in this namespace. Egress means 
+	// any outgoing traffic from pods, whether that be to other pods within this namespace
+	// or any destination outside of this namespace.
+	DefaultDeny EgressIsolationPolicy = "DefaultDeny"
```

Comment:
Constants aren't namespaced by type, so this has the same name as the DefaultDeny IngressIsolationPolicy (and therefore won't compile). Maybe just rename the type to IsolationPolicy?